ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10-upgrade-from-eus-4.8.yaml

```diff
@@ -23,8 +23,6 @@ tests:
   interval: 48h
   steps:
     cluster_profile: aws-2
-    dependencies:
-      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
```

Comment:
So if we have to remove OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE, we have to use RELEASE_IMAGE_LATEST as the image for our initial installation, is that right?

Reply:
After checking more steps in step-registry, it turns out that things are more complex than I have thought.
Some installation are using `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`, and some of them use `RELEASE_IMAGE_LATEST`.  Also some steps use OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE from `release:initial`, and some steps use it from `release:latest`.

Reply:
e.g, in upi azure installation,
```
if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
  exit 1
fi
echo "Installing from release ${RELEASE_IMAGE_LATEST}"
```

while in upi aws installation, it installs from OVERRIDE without any check.
```
echo "Installing from initial release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
```

Reply:
😞 These steps are implemented by different teams and are not consistently using the environment variables...

Reply:
@jhou1  To workaround the issue, tried https://github.com/openshift/release/pull/26196 and it works.

The idea is that,
point both `initial` and `latest` to stable 4.8, and introduce a new env `target` to point to 4.10.